### PR TITLE
Examples docs #74

### LINKED ISSUE #74: Put examples of use into an examples.rst after Quick Start

### PR DESCRIPTION
**Pull request by numbat**
Fixes #74

### CODE REVIEW

File: docs/keystore.rst [hunk 1]
```diff
@@ -16,16 +16,34 @@
 Key Store
 =========
 
-The ``DeltaKeyStore`` provides the interface for a key-storage
-backend of choice.
+The management and storage of private keys is the responsibility of the
+client. The ``DeltaKeyStore`` provides the interface for a key-storage
+implementation. The ``FileSystemKeyStore`` is an implementation to store keys
+on PEM formats on the file system.
```
Comment (paulb-covata) [inline docs/keystore.rst]: in PEM formats

File: docs/example_use.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+.. code:: python
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+
+Identity
+--------
+
+- Create an identity
```
Comment (myuwono) [inline docs/example_use.rst]: "Creating" an identity - just to be consistent with 'initialising' above ^^

File: docs/example_use.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+These examples assume a folder called ``~/keystore`` is present with
+``passPhrase`` as the password. Each example code-snippet is self-contained
+and runnable.
+
+Initialisation
+--------------
+- Initialising SDK Client (Configuration via Parameters)
+
+.. code:: python
+
```
Comment (myuwono) [inline docs/example_use.rst]: missing imports

File: docs/example_use.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+   from covata import delta
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   client.create_identity()
+
+- Get your own identity
```
Comment (myuwono) [inline docs/example_use.rst]: "Getting"

File: docs/example_use.rst [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+.. code:: python
+
+   from covata import delta
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   identity = client.get_identity("8e91cb8c-1ea5-4b69-bedf-9a14940cce44")
+
+- Get a different identity
```
Comment (myuwono) [inline docs/example_use.rst]: Getting~

Comment (paulb-covata) [inline docs/example_use.rst]: Retrieving?  Getting sounds a bit odd

File: docs/example_use.rst [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+.. code:: python
+
+   from covata import delta
```
Comment (myuwono) [inline docs/example_use.rst]: I feel the way it is done right now makes our doco prone to mistakes as there's no direct link to the code. why not use literalinclude instead? this way we can put each example in one .py file which has IDE support for linting, syntax highlighting and code completion.

```
- Getting a different identity

  .. literalinclude:: examples/get_different_identity.py
```

File: docs/example_use.rst [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   identity = client.get_identity("8e91cb8c-1ea5-4b69-bedf-9a14940cce44",
+                                  "1cb9375f-329c-405a-9b0c-b1659d9c66a4")
+
+- Add metadata to an identity
+
```
Comment (myuwono) [inline docs/example_use.rst]: missing example

File: docs/example_use.rst [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+- Create an identity
+.. code:: python
+
+   from covata import delta
+
+   key_store = FileSystemKeyStore("~/keystore/", "passPhrase")
+   client = Client(key_store)
+
+   client.create_identity()
```
Comment (myuwono) [inline docs/example_use.rst]: identity can also be created with optional metadata and external id

File: docs/example_use.rst (hunk 5 above)
Comment (myuwono) [inline docs/example_use.rst]: Yep! i guess it depends if we want to settle with <verb>-ing or not? better make it consistent

File: docs/example_use.rst (hunk 7 above)
Comment (numbat) [inline docs/example_use.rst]: Yes - missing functionality - I'll remove this for now.

File: docs/glossary.rst [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+   Delta so that the service can verify request authenticity and ownership.
+   The private signing key must be managed outside of Delta.
+
+Secret
+   An entry in Delta, comprising of protected secret content, encryption
+   details, core attributes, and secret metadata.
+
+SecretId
+   An identifier generated by Delta that is unique and associated with
+   retrieval and designation of an Secret.
```
Comment (ceven) [inline docs/glossary.rst]: a Secret? or is there a reason to use 'an'?

File: docs/identity.rst [hunk 10]
```diff
@@ -15,6 +15,28 @@
 Identity
 ========
 
+An Identity is entity (such as user, device, or another service) registered
```
Comment (ceven) [inline docs/identity.rst]: 'an' entity
Missing 'an'

File: docs/example_use.rst [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+   :linenos:
+   :emphasize-lines: 8, 11-12, 15
+
+-  Deleting a secret
+
+.. literalinclude:: ../examples/snippets/deleting_a_secret.py
+   :language: python
+   :lines: 15-
+   :linenos:
+   :emphasize-lines: 8, 11-13
```
Comment (myuwono) [inline docs/example_use.rst]: Superrrr cool bro 😎

**Comment by myuwono** [comment]
Looks good man

**Pull request merged**